src/naquibalam/Test1_NonRepeat.java

```diff
@@ -0,0 +1,29 @@
+package naquibalam;
+
+public class Test1_NonRepeat {
+	
+	char getFirstNonRepetingChar(String input) {
+		char returnChar = 0;
+		for (int i = 0; i <= input.length() - 1; i++) {
+			boolean repeatingFound = false;
+			for (int j = 0; j <= input.length() - 1; j++) {
```

Comment:
here you can use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop